src/app/modules/main/profile_section/wallet/networks/view.nim

```diff
@@ -46,6 +47,24 @@ QtObject:
     self.areTestNetworksEnabled = not self.areTestNetworksEnabled
     self.areTestNetworksEnabledChanged()
 
+  proc isSepoliaEnabledChanged*(self: View) {.signal.}
+
+  proc getIsSepoliaEnalbled(self: View): bool {.slot.} =
+    return self.isSepoliaEnabled
+
+  QtProperty[bool] isSepoliaEnabled:
+    read = getIsSepoliaEnalbled
+    notify = isSepoliaEnabledChanged
+
+  proc setIsSepoliaEnabled*(self: View, isSepoliaEnabled: bool) =
+    self.isSepoliaEnabled = isSepoliaEnabled
+    self.isSepoliaEnabledChanged()
+
+  proc toggleIsSepoliaEnabled*(self: View) {.slot.} =
+    self.delegate.toggleIsSepoliaEnabled()
+    self.isSepoliaEnabled = not self.isSepoliaEnabled
```

Comment:
@alaibe remove this and the line below, cause that will update the toggle on the UI regardless of the real change result.

Instead, you should try to connect to `SIGNAL_USE_OF_SEPOLIA_NETWORK_UPDATED` signal and update the UI based on the value of the `value` argument `self.view. setIsSepoliaEnabled(args.value)`